src/main/java/com/exactpro/th2/check1/Check1Handler.java

```diff
@@ -174,4 +185,64 @@ public void submitNoMessageCheck(NoMessageCheckRequest request, StreamObserver<N
             responseObserver.onCompleted();
         }
     }
+
+    @Override
+    public void submitMultipleRules(MultiRulesRequest request, StreamObserver<MultiRulesResponse> responseObserver) {
+        if (logger.isInfoEnabled()) {
+            logger.info("Rules list for request '{}' started", MessageUtils.toJson(request));
+        }
+
+        final boolean isResponseNeeded = !request.getOmitResponse();
+        final MultiRulesResponse.Builder responseBuilder = MultiRulesResponse.newBuilder();
```

Comment:
In all our projects we don't use `final` for local variables. Probably, we should follow the common conversion and don't start doing that here